baseapp/baseapp_test.go

```diff
@@ -270,12 +290,10 @@ type testTx struct {
 
 const msgType2 = "testTx"
```

Comment:
Could we add a comment here saying that the point of this is that its a sample message that can fail ValidateBasic?